src/java.base/share/classes/sun/security/util/KeyStoreDelegator.java

```diff
@@ -284,7 +289,18 @@ public void engineLoad(InputStream stream, char[] password)
             }
 
             if (debug != null) {
-                debug.println("Loaded a keystore in " + type + " format");
+                if (stream instanceof FileInputStream) {
+                    String keystorePath = SharedSecrets
+                                    .getJavaIOFileInputStreamAccess()
+                                    .getPath((FileInputStream) stream);
+                    if (keystorePath != null) {
+                        debug.println("Loaded " + keystorePath.substring(
```

Comment:
looking at debug output - it might be useful to surround the filename with quotes to emphasize that name is being printed.  e.g.

Loaded "keystore" keystore in PKCS12 format